src/asset-swapper/utils/market_operation_utils/fills.ts

```diff
@@ -6,6 +6,19 @@ import { MarketOperation, NativeOrderWithFillableAmounts } from '../../types';
 import { DEFAULT_FEE_ESTIMATE, POSITIVE_INF, SOURCE_FLAGS } from './constants';
 import { DexSample, ERC20BridgeSource, FeeEstimate, FeeSchedule, Fill, MultiHopFillData } from './types';
 
+function toSourceFlagKey(type: FillQuoteTransformerOrderType): 'LimitOrder' | 'RfqOrder' | 'OtcOrder' {
+    switch (type) {
+        case FillQuoteTransformerOrderType.Limit:
+            return 'LimitOrder';
+        case FillQuoteTransformerOrderType.Rfq:
+            return 'RfqOrder';
+        case FillQuoteTransformerOrderType.Otc:
+            return 'OtcOrder';
+        default:
+            return 'LimitOrder';
```

Comment:
nit: should we just error out on `default`? 

perhaps we can rename it to `toNativeSourceFlagKey`.

Reply:
agree on the rename - but not sure about throwing an error just in case some (insane/works by accident) existing flow relies on `LimitOrder` being the fallback